node-runner-cli/setup/SystemDSetup.py

```diff
@@ -464,8 +464,8 @@ def install_systemd_service(settings: SystemDConfig, args):
     def chown_files(settings):
         run_shell_command(['sudo', 'chown', 'radixdlt:radixdlt',
                            f'{settings.core_node.keydetails.keyfile_path}/{settings.core_node.keydetails.keyfile_name}'])
-        run_shell_command(['sudo', 'chown', 'radixdlt:radixdlt',
-                           f'{settings.common_config.genesis_bin_data_file}'])
+        # run_shell_command(['sudo', 'chown', 'radixdlt:radixdlt',
```

Comment:
Make it conditionally so that we can run for testnets

Reply:
done